windows/web/slimevr_web_installer.nsi

```diff
@@ -378,6 +378,43 @@ Section "SlimeVR Server" SEC_SERVER
     WriteUninstaller "$INSTDIR\uninstall.exe"
 SectionEnd
 
+Section "Webview2" SEC_WEBVIEW
+    SectionIn RO
+    Var /GLOBAL webviewKeyA
+    Var /GLOBAL webviewKeyB
+
+
+    # detecting webview from the windows registry
+    ${If} ${RunningX64}
+        ReadRegStr $webviewKeyA HKLM "SOFTWARE\WOW6432Node\Microsoft\EdgeUpdate\Clients\{F3017226-FE2A-4295-8BDF-00C3A9A7E4C5}" "pv"
+        ReadRegStr $webviewKeyB HKCU "Software\Microsoft\EdgeUpdate\Clients\{F3017226-FE2A-4295-8BDF-00C3A9A7E4C5}" "pv"
+    ${Else}
+        ReadRegStr $webviewKeyA HKLM "SOFTWARE\Microsoft\EdgeUpdate\Clients\{F3017226-FE2A-4295-8BDF-00C3A9A7E4C5}" "pv"
+        ReadRegStr $webviewKeyB HKCU "Software\Microsoft\EdgeUpdate\Clients\{F3017226-FE2A-4295-8BDF-00C3A9A7E4C5}" "pv"
```

Comment:
On my machine, it's located in `HKCU\Microsoft\EdgeUpdate\Clients\{56EB18F8-B008-4CBD-B6D2-8C97FE7E9062}`, and it doesn't have `pv` key. Does this work?

Reply:
https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution#detect-if-a-suitable-webview2-runtime-is-already-installed

Yes it should be pv